features/share/access-controls/user-access-controls/README.md

```diff
@@ -0,0 +1,265 @@
+<p align="center">
+  <a href="https://vendia.net/">
+    <img src="https://raw.githubusercontent.com/vendia/examples/main/vendia-logo.png" alt="vendia logo" width="100px">
+  </a>
+</p>
+
+# User Access Controls
+User access controls protect the user account management operations provided by Vendia Share.  This prevents unauthorized access to user management functions and enables features like self-service user management within an organization.
```

Comment:
```suggestion
You will use User Access Controls to prevent unauthorized access to user management functions and to enable features such as self-service user management within an organization.
```
```suggestion
User access controls protect the user account management operations provided by Vendia Share.  This prevents unauthorized access to user management functions and enables features like self-service user management within an organization.
```

Reply:
Try to merge the first and second sentences. They both seem to imply the same thing.  suggested a change.

Reply:
I'm using the first few sentences of each section to describe the feature, not what the user will do.  The intent is to provide them a bit of context before diving in to the "you will..." portion.  Let me know if that flow works.

Reply:
the new version of the definition looks great. I noticed it because at all other places you kept it simple. This is the only place where I have to read 3-4 times to fully comprehend it.